### PR TITLE
Update compilers

### PR DESCRIPTION
**Pull request by VoR0220**
* solves eris-ltd/eris-cli#1100
* solves eris-ltd/eris-cli#997 and thus lays the base to solve eris-ltd/eris-cli#1054

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -1,46 +1,89 @@
...
+  version: 8ca15f5455104403db4202c995e2f6e161654c02
+  subpackages:
+  - go/docs
+- name: github.com/eris-ltd/eris-cli
+  version: 002c63e2d0850b33c279c29fee52dd30853e100d
   subpackages:
-  - go/common
+  - config
+  - version
 - name: github.com/eris-ltd/eris-logger
```
Comment (zramsay) [inline glide.lock]: should be gone, no?

File: glide.yaml [hunk 2]
```diff
@@ -1,9 +1,10 @@
 package: github.com/eris-ltd/eris-compilers
 import:
-- package: github.com/ebuchman/go-shell-pipes
 - package: github.com/eris-ltd/common
```
Comment (zramsay) [inline glide.yaml]: also needs to go

File: perform/perform_compilation.go [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+	cached := CheckCached(request.Includes, request.Language)
+
+	log.WithField("cached?", cached).Debug("Cached Item(s)")
+
+	/*for k, v := range request.Includes {
+		log.WithFields(log.Fields{
+			"k": k,
+			"v": string(v.Script),
+		}).Debug("check request loop")
+	}*/
```
Comment (zramsay) [inline perform/perform_compilation.go]: remove or uncomment

File: glide.lock (hunk 1 above)
Comment (VoR0220) [inline glide.lock]: hmmm...indeed it should be. Will need to refactor the yaml and the lock.

File: glide.yaml (hunk 2 above)
Comment (VoR0220) [inline glide.yaml]: indeed.

File: perform/perform_compilation.go [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+	}
+}
+
+func runCommand(tokens ...string) (string, error) {
+	cmd := tokens[0]
+	args := tokens[1:]
+	shellCmd := exec.Command(cmd, args...)
+	output, err := shellCmd.CombinedOutput()
+	s := strings.TrimSpace(string(output))
+	return s, err
```
Comment (VoR0220) [inline perform/perform_compilation.go]: ++

File: perform/perform_compilation.go (hunk 3 above)
Comment (zramsay) [inline perform/perform_compilation.go]: it bloats the code for all others reading it. either the debug levels aren't tuned appropriately or it isn't used enough to be kept around.

**Pull request merged**